rapids-4-spark-tools/src/main/scala/com/nvidia/spark/rapids/tool/profiling/CollectInformation.scala

```diff
@@ -113,4 +112,17 @@ class CollectInformation(apps: ArrayBuffer[ApplicationInfo], fileWriter: FileWri
       }
     }
   }
+
+  // Print SQL Plan Metrics
+  def printSQLPlanMetrics(shouldGenDot: Boolean, outputDir: String,
+      writeOutput: Boolean = true): Unit ={
+    for (app <- apps){
+      val messageHeader = "\nSQL Plan Metrics for Application:\n"
+      val accums = app.runQuery(app.generateSQLAccums, fileWriter = Some(fileWriter),
+        messageHeader=messageHeader)
+      if (shouldGenDot) {
```

Comment:
this is probably no longer needed because we have a different path now for generating dot files

Reply:
we can merge and remove in a follow on if that is easier

Reply:
I moved it here, what else is htere?

Reply:
https://github.com/NVIDIA/spark-rapids/pull/2590/files#diff-efee456af06a26048911a4bba2094682835a780bd709bbf4572d27cd782e6339L128

Reply:
here we can reuse accums

Reply:
ah, ok. I hadn't realized you had moved it